demo-app/src/components/aruba/ArubaPopup.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Box, Text } from 'grommet';
+import { STATUS_MAP, Popup } from 'grommet-leaflet-core';
+
+const StatusSummary = ({ status, location }) => {
+  const StatusIcon = STATUS_MAP[status]?.icon;
+  return (
+    <Box align="center" direction="row" gap="xsmall">
+      <StatusIcon color={STATUS_MAP[status].color} size="small" />
+      <Text color="text-strong" size="xsmall">
+        {location.status[status]} {status}
+      </Text>
+    </Box>
+  );
+};
+
+export const ArubaPopup = ({ location }) => {
+  return (
+    <Popup>
```

Comment:
Small comment. But I wonder if the caller shouldn't have to import "Popup" and this should just come for free with the `popup` prop. Similar to what we have in MarkerCluster.

Reply:
Yes, good idea! Just pushed up that change